<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>549</width>
     <height>433</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background : #4e596f;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">padding-top: 0px;
margin-left: 40px;
margin-right: 40px;
margin-bottom: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_changeProfile">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>65</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>edit profile icon</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: #f76163;
border: 2px solid white;
border-radius: 30px;
}

QPushButton:hover{
background-color: #4e596f;
}

QPushButton:pressed{
background-color: #242A38;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources/res.qrc">
           <normaloff>:/res/profile-pics/default.png</normaloff>:/res/profile-pics/default.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>59</width>
           <height>59</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="uname">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_uname">
           <property name="styleSheet">
            <string notr="true">QLabel{
	text-decoration: none;
	margin-left: 0px;
	margin-bottom: 0px;
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledit_uname">
           <property name="styleSheet">
            <string notr="true">background: white;
border:1px solid white;
border-radius: 9px;
padding: 1 8px;
margin: 0 0 20 0px;
color: #242A38;
font: &quot;Segoe UI Emoji&quot;;
selection-background-color: #f76163;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="pwd">
         <item>
          <widget class="QLabel" name="label_pwd">
           <property name="styleSheet">
            <string notr="true">QLabel{
	text-decoration: none;
	margin-left: 0px;
	margin-bottom: 0px;
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledit_pwd">
           <property name="styleSheet">
            <string notr="true">background: white;
border:1px solid white;
border-radius: 9px;
padding: 1 8px;
margin: 0 0 20 0px;
color: #242A38;
font: &quot;Segoe UI Emoji&quot;;
selection-background-color: #f76163;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="cnfrmPwd">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_cnfrmPwd">
           <property name="styleSheet">
            <string notr="true">QLabel{
	text-decoration: none;
	margin-left: 0px;
	margin-bottom: 0px;
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Confirm Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_cnfrmPwd">
           <property name="styleSheet">
            <string notr="true">background: white;
border:1px solid white;
border-radius: 9px;
padding: 1 8px;
margin: 0 0 20 0px;
color: #242A38;
font: &quot;Segoe UI Emoji&quot;;
selection-background-color: #f76163;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showPwds">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show/hide all the passwords&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox {
    spacing: 5px;
	font-size: 11pt;
	color:white;
	margin-bottom: -3px;
	margin-left: 0px;
}

QCheckBox::indicator {
    width: 13px;
	height: 17px;
}

QCheckBox::indicator:unchecked {
    image: url(:/res/checkbox.png);
}

QCheckBox::indicator:checked {
    image: url(:/res/checkbox_checked.png);
}
</string>
         </property>
         <property name="text">
          <string>show passwords</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="buttons">
         <item>
          <widget class="QPushButton" name="btn_signup">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save all changes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background: #f76163;
    border-width: 1px;
    border-radius: 13px;
    border-color: #f76163;
    font: 12pt;
    color: white;
    padding: 10px;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 15px;
	margin-bottom: 10px;
	min-height: 20px;
}

QPushButton:hover {
background: qradialgradient(cx:0.3, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 white);
}

QPushButton:pressed {
background: qradialgradient(cx:0.7, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 black);
}</string>
           </property>
           <property name="text">
            <string>SIGNUP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Abort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: #242A38;
    border-width: 1px;
    border-radius: 13px;
    border-color: #242A38;
    font: 12pt;
	color: white;
    padding: 10px;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px;
	min-height: 20px;
}

QPushButton:hover {
background: qradialgradient(cx:0.3, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #242A38, stop:1 white);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(cx:0.7, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #242A38, stop:1 #000);
}</string>
           </property>
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
